decompile/preprossing/standardize.py

```diff
@@ -0,0 +1,33 @@
+"""Standardize assembly output from objdump command."""
+import re
+
+
+def standardize_asm_file(asm_file_path: str) -> str:
+    """Standardize asm file.
+
+    Args:
+        asm_file_path(str): path to asm file.
+
+    Returns:
+        str: standardized asm file as text."""
+    section_start_text = "Disassembly of section"
+    memory_address_format_example = "48 c7 45 f8 00 00 00    "
```

Comment:
Write a comment explaining why are you using this `memory_address_format_example` for.

Reply:
Noted.